### PR TITLE
Add Hyper-V queries example

### PR DESCRIPTION
**Pull request by JDA88**

### CODE REVIEW

**Comment by JDA88** [comment]
New to Github... Learning the hard way :)

File: docs/collector.hyperv.md [hunk 1]
```diff
@@ -96,7 +96,18 @@ Name | Description | Type | Labels
 _This collector does not yet have explained examples, we would appreciate your help adding them!_
 
 ## Useful queries
-_This collector does not yet have any useful queries added, we would appreciate your help adding them!_
+Percent of physical CPU ressources used per VM (on instance "localhost")
+```
+(sum (rate(windows_hyperv_vm_cpu_hypervisor_run_time{instance="localhost"}[1m]))) / ignoring(vm) group_left max (windows_cs_logical_processors{instance="localhost"}) / 100000
```
Comment (carlpett) [inline docs/collector.hyperv.md:101]: This "magic constant" 100000, do we know why it is required? Could it be a bug in the hyperv collector?

File: docs/collector.hyperv.md [hunk 2]
```diff
@@ -96,7 +96,18 @@ Name | Description | Type | Labels
 _This collector does not yet have explained examples, we would appreciate your help adding them!_
 
 ## Useful queries
-_This collector does not yet have any useful queries added, we would appreciate your help adding them!_
+Percent of physical CPU ressources used per VM (on instance "localhost")
```
Comment (carlpett) [inline docs/collector.hyperv.md:99]: Typo
```suggestion
Percent of physical CPU resources used per VM (on instance "localhost")
```

File: docs/collector.hyperv.md [hunk 3]
```diff
@@ -96,7 +96,18 @@ Name | Description | Type | Labels
...
+```
+(sum (rate(windows_hyperv_vm_cpu_hypervisor_run_time{instance="localhost"}[1m]))) / ignoring(vm) group_left max (windows_cs_logical_processors{instance="localhost"}) / 100000
+```
+Percent of physical CPU ressources used by all VMs (on all monitored hosts)
```
Comment (carlpett) [inline docs/collector.hyperv.md:103]: Typo
```suggestion
Percent of physical CPU resources used by all VMs (on all monitored hosts)
```

File: docs/collector.hyperv.md [hunk 4]
```diff
@@ -96,7 +96,18 @@ Name | Description | Type | Labels
...
+```
+(sum by (instance)(rate(windows_hyperv_vm_cpu_total_run_time{}[1m]))) / max by (instance)(windows_cs_logical_processors{}) / 100000
+```
+Percent of physical CPU ressources by the Hosts himself (on all monitored hosts)
```
Comment (carlpett) [inline docs/collector.hyperv.md:107]:
```suggestion
Percent of physical CPU resources by the hosts themselves (on all monitored hosts)
```

**Comment by carlpett** [comment]
Regarding the DCO check, you need to add a signoff on *all* the commits, right now it's only there on the last one. 
You can do `git rebase --signoff master` to add it to all the commits on your branch.

**Comment by JDA88** [comment]
Oups, sorry about that. Do I have to do another thing to have this validated? First time on a open GitHub project a little bit lost :)

I did :
```
git rebase --signoff master
Current branch master is up to date, rebase forced.
Successfully rebased and updated refs/heads/master.
```

File: docs/collector.hyperv.md (hunk 1 above)
Comment (JDA88) [reply docs/collector.hyperv.md:101]: It's probably related to the time format. Some king of Hypervisor Tick... I already saw 1000 or 10000 on some places, but 100000 never.
Poor documentation on those from MS :)

**Comment by carlpett** [comment]
No worries! Great that you've joined and are helping! :)
I think I see in your command output the reason it doesn't work like I expected, at least. The usual workflow is that'd you'd make a branch in your forked repo, but in your case you are working directly on the `master` branch (easily done if you're new to git). I think I'll avoid trying to explain how rebase works in this comment (there are a lot of good tutorials on the web), but in short, you'll want to do something similar to `git rebase --signoff origin/master`. The specifics depend on how you've set up your local clone. If you run `git remote -v`, what output do you get?

**Comment by JDA88** [comment]
I'm sure I have made a mistake somewhere, I never used Git before, and I thought it would not be that hard to edit one doc file…
First mistake I made was working on the `master`. But I got an “obviously” access denied on the commit. Then second try I made a fork and worked on this. I though it was the thing to do from what I say on the internet but I’m probably still missing something.

I get this on the `git remote -v` command:
```
origin  https://github.com/JDA88/windows_exporter (fetch)
origin  https://github.com/JDA88/windows_exporter (push)
```

**Comment by carlpett** [comment]
Yes, git's initial learning curve is a bit steep unfortunately.
Forking was the right thing to do, but in the future you'll have a much easier time if you create a branch. Here's a few steps that should fix the current DCO situation:
- Create a new remote pointing to this repo: `git remote add upstream https://github.com/prometheus-community/windows_exporter`
- Rebase from the new remote: `git rebase --signoff upstream/master`

File: docs/collector.hyperv.md (hunk 1 above)
Comment (carlpett) [reply docs/collector.hyperv.md:101]: There's a factor 10,000,000 that occurs in a couple of similar places for ticks-to-seconds conversions. But let's leave that out of scope for this PR :)

**Comment by JDA88** [comment]
> Yes, git's initial learning curve is a bit steep unfortunately.
Haha that's an understatement, I’ve been working on multiple Source Control on the past 30 years and I’ve never felt so noob / frustrated...

First command did not return anything (as expected) .
Second one did not worked:
```
git rebase --signoff upstream/master
fatal: invalid upstream 'upstream/master'
```

**Comment by carlpett** [comment]
Ah, forgot a crucial bit there (just to drive the ease of use home :joy:), you'll need to run `git fetch upstream` before the rebase command, otherwise git doesn't know about the branches in the remote.

**Comment by JDA88** [comment]
This one look more promising `git fetch upstream`
```
remote: Enumerating objects: 2, done.
remote: Counting objects: 100% (2/2), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 2 (delta 0), reused 0 (delta 0), pack-reused 0
Unpacking objects: 100% (2/2), 1.22 KiB | 83.00 KiB/s, done.
From https://github.com/prometheus-community/windows_exporter
 * [new branch]      master     -> upstream/master
 * [new branch]      smart      -> upstream/smart
 * [new branch]      sql-server -> upstream/sql-server
```
Then 
```
git rebase --signoff upstream/master
Successfully rebased and updated refs/heads/master.
```

**Comment by carlpett** [comment]
:tada: Then do a `git push --force` and the DCO check should pass!

**Comment by JDA88** [comment]
Worked!! That was so simple :)
Thank again for your help. So next time you are telling me I just do a branch from the main Master? I can forget about a personal fork, correct?

**Comment by carlpett** [comment]
Great! Yes, working from a branch will make things much easier (for example, if there's a different change merged to master from another PR, it gets messy to sort it out otherwise). You'll still need a fork unless you have write access to the repo, though. It is sort of like a staging area for the changes you are proposing. The way I find works is having the local `master` branch track the upstream repo's `master`, and then whenever I want to make a change, I branch off that and push to my fork.

**Comment by JDA88** [comment]
I'll try to do better next time :). Anything else I have to do to get this change integrated in the main Master?

**Comment by carlpett** [comment]
Nope, I was just waiting for CI to pass, will merge now! Thanks a lot :)

**Pull request merged**